client/components/jetpack/threat-item-subheader/index.tsx

```diff
@@ -33,9 +33,41 @@ const formatDate = ( date: Date ) => {
 	return moment( date ).format( 'LL' );
 };
 
+const getThreatStatusMessage = ( translate, threat: Threat ) => {
+	const { status, fixedOn } = threat;
+
+	const date = fixedOn && formatDate( fixedOn );
+
+	if ( status === 'fixed' ) {
+		return date
+			? translate( 'Threat fixed on %(date)s', {
+					args: { date },
+					comment: 'Past tense action: a threat was fixed on a specific date',
+			  } )
+			: translate( 'Threat fixed', {
+					comment: 'Past tense action: a threat was fixed on an unspecified date',
+			  } );
+	}
+
+	if ( status === 'ignored' ) {
+		return date
+			? translate( 'Threat ignored on %(date)s', {
+					args: { date },
+					comment: 'Past tense action: a threat was ignored on a specific date',
+			  } )
+			: translate( 'Threat ignored', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Threat ignored.' )` *ES Score: 13*